The complaint is from flake8 and is:
```
+flake8 sympy
sympy/physics/units/util.py:161:5: F401 'functools' imported but unused
sympy/physics/units/util.py:162:5: F401 'operator' imported but unused
```
The complaint is that these functions should not be imported if they are no longer used.

If you install flake8 you can run this command locally as
```
$ flake8 sympy
```

--- a/sympy/physics/units/util.py
+++ b/sympy/physics/units/util.py
@@ -158,8 +158,6 @@ def check_dimensions(expr, unit_system="SI"):
     # Also, when doing substitutions, multiplicative constants
     # might be introduced, so remove those now
 
-    import functools
-    import operator
     from sympy.physics.units import UnitSystem
     unit_system = UnitSystem.get_unit_system(unit_system)
 

